packages/cells/src/widget.ts

```diff
@@ -1040,8 +1051,23 @@ export class CodeCell extends Cell<ICodeCellModel> {
       this.addClass(DIRTY_CLASS);
     }
 
-    this._outputPlaceholder = new OutputPlaceholder(() => {
-      this.outputHidden = !this.outputHidden;
+    this._outputPlaceholder = new OutputPlaceholder({
+      callback: () => {
+        this.outputHidden = !this.outputHidden;
+      },
+      text: (
+        ((this.model.toJSON().outputs as nbformat.IOutput[])[0]
+          ?.text as string) ?? ''
+      ).split('\n')[0]
```

Comment:
Not sure why but `this.model.outputs.get(0)` is returning undefined? Any idea why that might be happening?

Reply:
Update: `this.model.outputs.get(0)` actually is returning an `OutputModel` object, but `this.model.outputs.get(0)?.data` only has a field called `application/vnd.jupyter.stdout` - no `'text/plain'`.

Reply:
Just pushed with that field but let me know if you want me to update it!

Reply:
This is the problem with outputs; you don't know which mimetype will be available. I guess we should try the 3 following default text type:

https://github.com/jupyterlab/jupyterlab/blob/b69d2c3b91ac670b68f7bbf54b7d330060cd7e9c/packages/rendermime/src/factories.ts#L67-L73

If you could use directly `textRendererFactory.mimeTypes` it would be the best.